files/usr/share/jeos-firstboot/jeos-firstboot-functions

```diff
@@ -54,6 +66,7 @@ call_module() {
 
 call_module_hook() {
         local hook="$1"
+	_init_modules
```

Comment:
Should be called in global scope on inclusion, like before.

Reply:
moved it to after dryrun etc in main file. doing that on source seems ugly to me